maxi-blocks/maxi-blocks
# Issue 3109 - remove force responsive switch

# Description
Removed force responsive switch. Kept fit-content for svg-icon.


Fixes #3109 

# How to test?
Create svg-icon in container, set its width to a big number, check if it fits in container.


# Test checklist



**_Front/Back Testing_**

-   [x] Same on responsive

**_Pre-Code Testing_**

-   [x] Same on responsive

# Checklist
